.github/workflows/request_codeowners_review.yml

```diff
@@ -45,10 +51,21 @@ jobs:
           if [ -n "$reviewers" ]; then
             # Remove @ from reviewers (it stays in the beginning of the username)
             reviewers_cleaned=${reviewers//@/}
-            # Convert reviewers to comma separated list
-            reviewers_comma_separated=$(echo "$reviewers_cleaned" | tr ' ' ',')
-            echo "Requesting review from: $reviewers_comma_separated"
-            gh pr edit ${{ github.event.pull_request.number }} --add-reviewer $reviewers_comma_separated
+            # Convert the space-separated usernames into a Bash array
```

Comment:
Would it make sense to add a comment with a reference to the reported issue? And/or put that reference in the commit message?

Reply:
Yep, I'll link if it makes sense.
But the workflow still fails :(